components/mediation-commons/src/main/java/org/wso2/carbon/mediation/commons/rest/api/swagger/SwaggerConstants.java

```diff
@@ -260,4 +260,15 @@
      */
     public static String[] DEFAULT_PRODUCES = {CONTENT_TYPE_JSON, CONTENT_TYPE_XML};
 
+    /**
+     * Default produce mime types
+     */
+    public static String DEFAULT_HOST = "localhost";
+
+    /**
+     * Default produce mime types
+     */
+    public static int DEFAULT_PORT = 8290;
```

Comment:
The comment and the constant does not match

Reply:
fixed